src/IssueViz/IssueVisualizationControl/IssueVisualizationControl.xaml

```diff
@@ -315,13 +334,35 @@
                                        Style="{StaticResource BaseLocationItemTextBlockStyle}" 
                                        Text="{Binding Location.Location.Message}"/>
                             <vsImaging:CrispImage Grid.Column="2"
-                                                Height="16" Width="16"
-                                                Moniker="{x:Static vsCatalog:KnownMonikers.DocumentWarning}" 
-                                                Visibility="{Binding Location.Span, Converter={StaticResource SpanToInvertedVisibilityConverter}, Mode=OneWay}"/>
+                                                  Height="16" Width="16"
+                                                  Moniker="{x:Static vsCatalog:KnownMonikers.DocumentWarning}" 
+                                                  Visibility="{Binding Location.Span, Converter={StaticResource SpanToInvertedVisibilityConverter}, Mode=OneWay}"/>
                         </Grid>
                     </DataTemplate>
                 </ListBox.Resources>
             </ListView>
+
+            <Border Name="NavigabilityWarning"
+                    Grid.Row="2" 
```

Comment:
As we discussed, it is currently shown as a footer. Can easily be changed to top, by changing the grid row to 1.